homeworks/vladyslav.kovalov_kovalov/3-async/loader.js

```diff
@@ -0,0 +1,26 @@
+export class Loader {
+    constructor() {
+        this.containerEl = document.querySelector('[data-posts-section]');
+        this.templateElement = document.querySelector('[data-loader-template]');
+    }
+
+    create() {
+        const content = this.templateElement.content.cloneNode(true);
+        const element = content.querySelector('[data-loader]');
+        return element;
+    }
+
+    add(element) {
+        this.containerEl.append(element);
+    }
```

Comment:
```suggestion
    show() {
        const content = this.templateElement.content.cloneNode(true);
        const element = content.querySelector('[data-loader]');
        this.containerEl.append(element);
    }
```

і `display-none` можна не вішати. Ти всеодно видаляєш його з дома по закінченню завантаження